packages/notifications/src/styled/StyledAlert.spec.tsx

```diff
@@ -6,39 +6,293 @@
+      const { container } = render(<StyledAlert type={type} />);
 
-      expect(container.firstChild).toHaveClass('is-rtl');
+      expect(container.firstChild).toHaveStyleRule('color', typeHues[type] as string, {
+        modifier: css`
+          ${StyledTitle}
+        ` as any
+      });
     });
+  });
 
+  describe('validation', () => {
     it('should render success styling correctly', () => {
       const { container } = render(<StyledAlert type="success" />);
 
-      expect(container.firstChild).toHaveClass('c-callout--success');
+      expect(container.firstChild).toHaveStyleRule(
+        'border-color',
+        DEFAULT_THEME.palette.green[300]
+      );
+      expect(container.firstChild).toHaveStyleRule(
+        'background-color',
+        DEFAULT_THEME.palette.green[100]
+      );
+      expect(container.firstChild).toHaveStyleRule('color', DEFAULT_THEME.palette.green[700]);
     });
 
     it('should render warning styling correctly', () => {
       const { container } = render(<StyledAlert type="warning" />);
 
-      expect(container.firstChild).toHaveClass('c-callout--warning');
+      expect(container.firstChild).toHaveStyleRule(
+        'border-color',
+        DEFAULT_THEME.palette.yellow[300]
+      );
+      expect(container.firstChild).toHaveStyleRule(
+        'background-color',
+        DEFAULT_THEME.palette.yellow[100]
+      );
+      expect(container.firstChild).toHaveStyleRule('color', DEFAULT_THEME.palette.yellow[700]);
     });
 
     it('should render error styling correctly', () => {
       const { container } = render(<StyledAlert type="error" />);
 
-      expect(container.firstChild).toHaveClass('c-callout--error');
+      expect(container.firstChild).toHaveStyleRule('border-color', DEFAULT_THEME.palette.red[300]);
+      expect(container.firstChild).toHaveStyleRule(
+        'background-color',
+        DEFAULT_THEME.palette.red[100]
+      );
+      expect(container.firstChild).toHaveStyleRule('color', DEFAULT_THEME.palette.red[700]);
     });
 
     it('should render info styling correctly', () => {
       const { container } = render(<StyledAlert type="info" />);
 
-      expect(container.firstChild).toHaveClass('c-callout--info');
+      expect(container.firstChild).toHaveStyleRule('border-color', DEFAULT_THEME.palette.grey[300]);
+      expect(container.firstChild).toHaveStyleRule(
+        'background-color',
+        DEFAULT_THEME.palette.grey[100]
+      );
+      expect(container.firstChild).toHaveStyleRule('color', DEFAULT_THEME.palette.grey[600]);
+    });
+
+    it(`should render the correct styling for Success Alert's close button`, () => {
+      const { container } = render(<StyledAlert type="success" />);
+      const { colors, palette } = DEFAULT_THEME;
+      const { successHue } = colors;
+
+      // Close style
+      expect(container.firstChild).toHaveStyleRule('color', palette[successHue][500], {
+        modifier: css`
+          ${StyledClose}
+        ` as any
+      });
```

Comment:
I wrote a few tests which assert that the styling for `<Alert>` and `<Notification>` descendants are rendered correctly. I think these tests provide good value because the styling is heavily guarded by logic. These tests are likely more valuable when the project gains more contributors.

Reply:
imo, it's smart to keep some basic testing in isolate around the styled (view) components.